WGS_pipeline/WGS_pipeline.wdl

```diff
@@ -0,0 +1,331 @@
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/BamToUnmappedRGBams.wdl" as BamToUnmappedRGBams 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/ArrayOfFilesToTxt.wdl" as ArrayOfFilesToTxt 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/PreProcessingForVariantDiscovery_GATK4.wdl" as PreProcessingForVariantDiscovery_GATK4 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/CNV_Somatic_Workflow_on_Sample.wdl" as CNV_Somatic_Workflow_on_Sample 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/cnn-variant-filter.wdl" as cnn_variant_filter 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGS_pipeline/Manta_SomaticSV.wdl" as Manta_SomaticSV 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGSmut_pipeline/CGA_WES_CCLE_Characterization_Pipeline_v0.1_Jul2019_copy.wdl" as CGA_WES_CCLE_Characterization_Pipeline_v0 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGSmut_pipeline/common_variant_filter.wdl" as common_variant_filter 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/WGSmut_pipeline/filterMAF_on_CGA_pipeline.wdl" as filterMAF_on_CGA_pipeline 
+
+
+workflow WGS_pipeline {
+
+	#BamToUnmappedRGBams
+	File ref_fasta
+	File ref_fasta_index
+
+	File input_bam
+	File input_bam_index
+	
+	String picard_path
+	String picard_docker
+
+	Int preemptible_tries
+
+	#ArrayOfFilesToTxt
+	String sample_name #this.name
+
+	#PreProcessingForVariantDiscovery_GATK4
+	String ref_name
+
+	File ref_dict
+
+	File dbSNP_vcf
+	File dbSNP_vcf_index
+
+	Array[File] known_indels_sites_VCFs
+	Array[File] known_indels_sites_indices
+
+	String gotc_docker
+	String gatk_docker
+	String python_docker
+
+	String gotc_path
+	String gatk_launch_path
+
+	#CNVSomaticPairWorkflow
+	File common_sites
+	File intervals
+
+	#cnn_variant_filter
+
+	# Manta
+
+	#CGA_Production_Analysis_Workflow
+	File ref_fasta_H19
+	File ref_fasta_H19_index
+	File ref_fasta_H19_dict
+
+	call BamToUnmappedRGBams.BamToUnmappedRGBamsWf as BamToUnmappedRGBamsWf {
+		input:
+			ref_fasta=ref_fasta,
+			ref_fasta_index=ref_fasta_index,
+			input_bam=input_bam,
+			picard_path=picard_path,
+			picard_docker=picard_docker,
+			preemptible_tries=preemptible_tries 
+	}
+
+	call ArrayOfFilesToTxt.CreateTxt as CreateTxt {
+		input:
+			array_of_files=BamToUnmappedRGBamsWf.sortsam_out,
+			list_name=sample_name
+	}
+
+	call PreProcessingForVariantDiscovery_GATK4.PreProcessingForVariantDiscovery_GATK4 as PreProcessingForVariantDiscovery_GATK4 {
```

Comment:
is this the HaplotypeCaller?

Reply:
no this is our mapping workflow

Reply:
its output is fed to cnn_variant_filter. I thought that is for filtering germlines. So which tool is doing the germline calling then?

Reply:
cnn_variant_filter. that is how this tool is called by GATK. it does both calling and filtering